### DISCUSSION TITLE
Osb-studio

### DISCUSSION THREAD

**Discussion by helio58** [tteck/Proxmox#3013] [Ideas]
Hi is it possible to create a script to run OBS ?

I have try with your Jellyfin script but I m not able to have it running HW 
Thanks

**Comment by bramsuurdje** [comment]
you do mean a RTMP server?

**Reply by helio58** [reply]
No, i mean the app  , i manage to install in the jellyfin lxc , removed jellyfin install obs-studio and install ubuntu-desktop- minimal. When runing obs the cpu is maxed . The gpu vaapi works with jellyfin . I havent manage to having it working with OBS.
